### ISSUE TITLE
UI Todos

### ISSUE THREAD

**Issue by yyx990803** [vuejs/vue-cli#1382]
Some notes after trying it out from an actual Yarn global install:

- [x] Missing dep `launch-editor` (fixed in 3bcc511)
- [x] `Error: Failed to resolve plugin: @vue/cli-service` when creating a new project. For some reason, `node_modules` is empty with `yarn.lock` present.
- [x] Some UI default values are not in-sync with the CLI (e.g. when choosing TypeScript, "Use class component" should default to true but it's not in UI, same for "Lint on Save")
- [x] UI should show disconnected state if user terminates the process from the command line

**Comment by zigomir** [comment]
`Error: Failed to resolve plugin: @vue/cli-service` is not directly related to UI imho. I'm using npm 6.1.0 and I can't do `vue create <project>` with `@vue/cli` 3.0.0-beta.12

**Comment by Akryum** [comment]
@zigomir Try unlinking any @vue related packages and remove `@vue/cli` it from global packages and then reinstall it. I just try and the cli command should still work.

**Comment by zigomir** [comment]
@Akryum I first tried with `npm uninstall -g @vue/cli` and then even `rm -rf @vue` from node's global `lib/node_modules`, but I still have same issue.

**Comment by Akryum** [comment]
Could you try with yarn just in case?

**Comment by Akryum** [comment]
Also could you create a new issue? Let's not spam this one. 😸

**Comment by Akryum** [comment]
> Some UI default values are not in-sync with the CLI (e.g. when choosing TypeScript, "Use class component" should default to true but it's not in UI, same for "Lint on Save")

@yyx990803 The `prompts.js` files don't have the default values set, is it intentional?

**Comment by zigomir** [comment]
Installing with `yarn add global @vue/cli` and then running `vue create` still fails with same error. I'm pasting whole error for more context:

```
 ERROR  Error: Failed to resolve plugin: @vue/cli-service
Error: Failed to resolve plugin: @vue/cli-service
    at Creator.resolvePlugins (/Users/ziga/.nvm/versions/node/v8.9.4/lib/node_modules/@vue/cli/lib/Creator.js:303:15)
    at Creator.create (/Users/ziga/.nvm/versions/node/v8.9.4/lib/node_modules/@vue/cli/lib/Creator.js:146:32)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by Akryum** [comment]
@zigomir Please create a new issue, thanks!

**Comment by Akryum** [comment]
All the notes fixed in #1386
